opta/core/gcp.py

```diff
@@ -222,6 +222,12 @@ def get_cluster_env(self) -> Tuple[str, str]:
         googl_provider = self.layer.root().providers["google"]
         return googl_provider["region"], googl_provider["project"]
 
+    def get_kubectl_context_name(self) -> str:
```

Comment:
this is a nice abstraction - anyway we can add a unit test just on this method? `test_kubectl_context_name` (aws/gcp)

Reply:
it's implementations are so small, that not really without mocking most of it